src/CartingService/Program.cs

```diff
@@ -13,9 +14,12 @@
 
 builder.Services.AddAutoMapper(Assembly.GetExecutingAssembly());
 builder.Services.AddMediatR(Assembly.GetExecutingAssembly());
-builder.Services.AddScoped<IApplicationDbContext, ApplicationDbContext>();
+builder.Services.AddSingleton<IApplicationDbContext, ApplicationDbContext>();
```

Comment:
Could you please explain, why you decided to have it as Singleton?

Reply:
LiteDb is a file-based DB. So, I don't want to open a connection each time when a request comes.